skin/base.css

```diff
@@ -105,3 +105,9 @@ statuspanel:-moz-locale-dir(ltr):not([mirror]) .statuspanel-inner {
 #verticaltabs-box .private-browsing-indicator {
   display: none;
 }
+
+#TabsToolbar[brighttext] .tabs-newtab-button,
+#TabsToolbar[brighttext] > #new-tab-button,
+#TabsToolbar[brighttext] > toolbarpaletteitem > #new-tab-button {
```

Comment:
Does this also apply to the dark DevTools theme?  Cause if so, we should change line 94 for consistency, and if not, we should add `[devtoolstheme="dark"]` to these…